tests/e2e/index.spec.ts

```diff
@@ -66,4 +66,15 @@ test.describe("Index page", () => {
         await expect(await page.innerText(".leaflet-tooltip-pane")).toContain("Bengo");
         await expect(await page.innerText(".leaflet-tooltip-pane")).toContain("FOI: 0.0134");
     });
+
+    test("resetMapButton click navigates to home page from country URL", async ({ page }) => {
+        await page.waitForURL(/dengue\/may24\/FOI/i);
+        const allRegions = await page.locator(GEOJSON_SELECTOR);
+        await expect(await allRegions).toHaveCount(1833);
+        await page.goto("/dengue/may24/serop9/MWI");
+        await expect(await allRegions).toHaveCount(1833);
```

Comment:
This should be a higher value than for home view, now that MWI is selected. This will be passing because it's 1833 immediately I guess.

Reply:
Well caught. Updated.